src/renderer/pages/settings-page.component.tsx

```diff
@@ -137,6 +144,27 @@ export function SettingsPage() {
         i18nService.setLanguage(item.value);
     };
 
+    const handleVersionClick = async () => {
+        let isChangelogResolved = false;
+
+        const changelogPromise = autoUpdater.showChangelog(await lastValueFrom(autoUpdater.getAppVersion()))
```

Comment:
no need to put the promise in a const here, just await the promise directly under the setTimout

Reply:
If you wanted to run it not imediatly you are not doing it right